assets/global.js

```diff
@@ -580,6 +580,37 @@ class ModalDialog extends HTMLElement {
 }
 customElements.define('modal-dialog', ModalDialog);
 
+class BulkModal extends HTMLElement {
+  constructor() {
+    super();
+  }
+
+  connectedCallback() {
+    const handleIntersection = (entries, observer) => {
+      if (!entries[0].isIntersecting) return;
+      observer.unobserve(this);
+      if (this.innerHTML.trim() === '') {
+        fetch(`${this.dataset.url}?section_id=bulk-quick-order-list`)
```

Comment:
@NathanPJF I cant seem to reproduce your error 🤔 Am I missing any settings you have in the collections page?

https://screenshot.click/17-10-m7nin-zu4of.png

Reply:
I checked your store, so 2 things were happening:
1. Your initial render doesn't have multiple `?` symbols in the URL. But as soon as you apply filters it does.
2. Your PDP page has the quick order list section enabled, so it didn't matter that the call wasn't rendering the exact section you wanted because the section was in the DOM eventually.

I just removed the quick order list from your PDP page and you'll see that you get the errors now: https://screenshot.click/17-09-kihlc-ws0rm.png

Reply:
Ah I see! I hadnt tested it out with the filters.

Very good catch thank you 👍 fixed! I went with approach #2 as I agree with you on #1